.github/workflows/export_category.yml

```diff
@@ -40,16 +40,15 @@ jobs:
           sudo apt update
           sudo apt install -y python3-pip python3-distutils
           sudo apt install -y gdal-bin
-          curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py | python3 -
+          curl -sSL https://install.python-poetry.org | python3 -
           curl -O https://dl.min.io/client/mc/release/linux-amd64/mc
           chmod +x mc
           sudo mv ./mc /usr/bin
           mc config host add spaces $AWS_S3_ENDPOINT $AWS_ACCESS_KEY_ID $AWS_SECRET_ACCESS_KEY --api S3v4
 
       - name: Poetry Install
         run: |
-          . $HOME/.poetry/env
-          pip install poetry
+          export PATH=$PATH:$HOME/.local/bin
           poetry install      
```

Comment:
```suggestion
          poetry install      
          poetry --version
```

Reply:
Suggesting as the Github Action log suggests this command [here](https://github.com/NYCPlanning/db-equitable-development-tool/runs/8209344747?check_suite_focus=true#step:4:330)

Reply:
We could have poetry print the version so we can see it in the logs, that seems like a fine suggestion